community/pulseaudio/APKBUILD

```diff
@@ -12,7 +12,7 @@ makedepends="m4 automake libtool intltool bash
 	dbus-dev libcap-dev alsa-lib-dev bluez-dev speexdsp-dev avahi-dev libressl-dev eudev-dev
 	libsndfile-dev gtk+3.0-dev json-c-dev fftw-dev gconf-dev sbc-dev jack-dev"
 install=""
-subpackages="$pkgname-dev $pkgname-doc $pkgname-libs $pkgname-bluez
+subpackages="$pkgname-openrc $pkgname-dev $pkgname-doc $pkgname-libs $pkgname-bluez
```

Comment:
This specifically already has -system (which is discouraged upstream), which takes the OpenRC scripts.

Reply:
Thanks, will fix it